src/Widgets/CameraView.vala

```diff
@@ -297,10 +297,12 @@ public class Camera.Widgets.CameraView : Gtk.Box {
                 "v4l2src device=%s name=%s num-buffers=1 !".printf (device_path, VIDEO_SRC_NAME) +
                 "videoscale ! video/x-raw, width=%d, height=%d !".printf (picture_width, picture_height) +
                 "videoflip method=%s !".printf ((horizontal_flip)?"horizontal-flip":"none") +
-                "videobalance brightness=%f contrast=%f !".printf (brightness_value.get_double (), contrast_value.get_double ()) +
+                "videobalance brightness=%s contrast=%s !".printf (
+                    convert_comma_to_dot (brightness_value.get_double ()),
```

Comment:
Maybe better to us https://valadoc.org/glib-2.0/string.delimit.html with `null` as argument so it uses the standard list of delimiters? That way hopefully covers more possible localization cases if there’s some other delimiters in use

Reply:
I am not sure how to use this function

Reply:
I think it would be like
```
"videobalance brightness=%f contrast=%f !".printf (brightness_value.get_double (), contrast_value.get_double ()).delimit (null, '.')
```

Reply:
Ah I guess `G_STR_DELIMITERS` doesn't include `,` so it would be like `.delimit (",", '.')`